.ci/teardown.sh

```diff
@@ -17,22 +17,50 @@
 # This script will get any execution log that may be useful for
 # debugging any issue related to Clear Containers.
 
-runtime_log_file="/var/lib/clear-containers/runtime/runtime.log"
+log_copy_dest=$1
 
-echo "Clear Containers Runtime Log:"
-sudo cat "$runtime_log_file"
+runtime_log_location="/var/lib/clear-containers/runtime/runtime.log"
+runtime_log_filename="cc-runtime.log"
+runtime_log_path="${log_copy_dest}/${runtime_log_filename}"
+proxy_log_filename="cc-proxy.log"
```

Comment:
Could you add blank lines between each block (runtime, proxy, shim, etc) to make this a little easier to read?

Reply:
done